cmd/podman/main.go

```diff
@@ -36,6 +38,19 @@ func main() {
 		return
 	}
 
+	shell := os.Getenv("SHELL")
+	if filepath.Base(os.Args[0]) == "podmansh" || strings.HasSuffix(os.Args[0], filepath.Base(shell)) {
+		var podmanshCommand = &cobra.Command{}
+		_, err := registry.NewContainerEngine(podmanshCommand, []string{"podmansh", "/bin/bash", "$*"})
+		if err != nil {
+			fmt.Errorf("Error running NewContainerEngine: %w", err)
+			return
+		}
+		err = containers.PodmanshExec(podmanshCommand, []string{"podmansh", "/bin/bash"})
```

Comment:
Should this be just sh?  /bin/bash is not available in some container images, (Alpine), but /bin/sh should always be there.